clang/test/CAS/fmodule-file-cache-key-errors.c

```diff
@@ -16,19 +16,19 @@
 // RUN:   -fcache-compile-job -Rcompile-job-cache &> %t/invalid.txt
 // RUN: cat %t/invalid.txt | FileCheck %s -check-prefix=INVALID
 
-// INVALID: error: option '-fmodule-file-cache-key' should be of the form <path>=<key>
+// INVALID: error: unknown argument: '-fmodule-file-cache-key=INVALID'
```

Comment:
Could we add a test here for the wrong number of arguments?

Reply:
Done.